runtime/doc/lua.txt

```diff
@@ -622,6 +622,10 @@ vim.api.{func}({...})					*vim.api*
         Example: call the "nvim_get_current_line()" API function: >
             print(tostring(vim.api.nvim_get_current_line()))
 
+vim.version.()    					*vim.version*
```

Comment:
shouldn't this be `vim.nvim_version`? (also stray period at the end)

Reply:
Fixed!